crypto/CryptoBot.py

```diff
@@ -24,7 +24,18 @@ def __init__(
         self.runMaintenance(5 * 60)
 
     def runMaintenance(self, interval: float) -> None:
-        self.trader.checkStops()
+        sales = self.trader.checkStops()
```

Comment:
maybe call this `executeStops` so it's more clear that this function has side effects